src/main/webapp/assets/modules/contest-detail/evaluation-criteria/challengeEvaluationCriteriaTab.js

```diff
@@ -0,0 +1,13 @@
+techlooper.directive('challengeEvaluationCriteriaInput', function () {
+    return {
+      restrict: "E",
+      replace: true,
+      templateUrl: "modules/contest-detail/evaluation-criteria/evaluationCriteriaInput.html",
+      scope: {
+        cri: "=",
+        contestDetail: "="
+      },
+      link: function (scope, element, attr, ctrl) {
```

Comment:
'ctrl' is defined but never used.<br>'attr' is defined but never used.<br>'element' is defined but never used.<br>'scope' is defined but never used.